TheMET/TheMET/ViewControllers/AboutAppViewController.swift

```diff
@@ -0,0 +1,137 @@
+            self.leftVersionLabel.leadingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.leadingAnchor, constant: 20),
+            self.leftVersionLabel.widthAnchor.constraint(equalTo: self.view.widthAnchor, multiplier: 0.5, constant: 0),
+            
+            self.leftVersionLabel.topAnchor.constraint(equalTo: self.imageView.bottomAnchor,constant: 30)
+        ])
+        NSLayoutConstraint.activate([
+            self.rightVersionLabel.leadingAnchor.constraint(equalTo: self.leftVersionLabel.trailingAnchor),
+            self.rightVersionLabel.trailingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.trailingAnchor, constant: -20),
+            self.rightVersionLabel.topAnchor.constraint(equalTo: self.imageView.bottomAnchor,constant: 30)
+        ])
+        NSLayoutConstraint.activate([
+            self.tableView.leadingAnchor.constraint(equalTo: self.view.leadingAnchor),
+            self.tableView.trailingAnchor.constraint(equalTo: self.view.trailingAnchor),
+            self.tableView.topAnchor.constraint(equalTo: self.leftVersionLabel.bottomAnchor,constant: 15),
+            self.tableView.heightAnchor.constraint(equalToConstant: 130 + 53.67)
+        ])
+        NSLayoutConstraint.activate([
+            self.copyRightLabel.leadingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.leadingAnchor, constant: 20),
+            self.copyRightLabel.trailingAnchor.constraint(equalTo: self.view.trailingAnchor),
+            self.copyRightLabel.topAnchor.constraint(equalTo: self.tableView.bottomAnchor, constant: 15)
+        ])
+        
+        self.tableView.rowHeight = UITableView.automaticDimension
+        self.tableView.register(AboutAppViewCell.self, forCellReuseIdentifier: AboutAppViewCell.aboutAppViewCellIdentifier)
+        self.tableView.dataSource = self
+        self.tableView.delegate = self
+    }
+    
+    private func addLabel(_ label : UILabel,color: UIColor?, title: String, textAlignment: NSTextAlignment) {
+        label.backgroundColor = .clear
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.apply(font: NSLocalizedString("serif_font", comment: ""), color: color, fontSize: 16, title: title)
+        label.textAlignment = textAlignment
+        self.view.addSubview(label)
+    }
+    
+    private func getAppVersion() -> String {
+        if let version = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String {
+            return version
+        } else {
+            return ""
+        }
+    }
+    
+    func showSafariPage(urlString: String) {
+        if let url = URL(string: urlString) {
+            let configuration = SFSafariViewController.Configuration()
+            configuration.entersReaderIfAvailable = true
+            let safariViewController = SFSafariViewController(url: url, configuration: configuration)
+            present(safariViewController, animated: true)
+        } else {
+            print ("error")
```

Comment:
Try to put some details of the error into the print. It is useless without a context,